### PR TITLE
chore: [IOBP-803] Add feedback banner definitions into new payment section

### PR DESCRIPTION
**Pull request by Hantex9**
## Short description
This PR adds definitions for the payment banner feedback adding some reusable types `Banner`, `BannerAction` and `LocalizedText` that provide all the needed properties to show a banner with an optional action.

## List of changes proposed in this pull request
- Added `feedbackBanner` under the `NewPaymentSection` definition with the provided min version to be visible
- Added `Banner` properties with the `BannerAction` and a reusable `LocalizedText` definition

### CODE REVIEW

File: status/backend.json [hunk 1]
```diff
@@ -77,6 +77,27 @@
...
+          "ios": "3.0.0.0",
+          "android": "3.0.0.0"
+        },
+        "title": {
+          "it-IT": "Puoi dirci com'è andata?",
+          "en-EN": "Puoi dirci com'è andata?"
+        },
+        "description": {
+          "it-IT": "Raccontaci la tua esperienza con il pagamento e aiutaci a migliorare.",
+          "en-EN": "Raccontaci la tua esperienza con il pagamento e aiutaci a migliorare."
```
[the remark this answers is not in the document]
Comment (Hantex9) [reply status/backend.json:92]: @thisisjp Please could you provide the English message to show for this banner?

File: status/backend.json [hunk 2]
```diff
@@ -77,6 +77,27 @@
...
+        },
+        "action": {
+          "label": {
+            "it-IT": "Vai al sondaggio",
+            "en-EN": "Vai al sondaggio"
```
[the remark this answers is not in the document]
Comment (Hantex9) [reply status/backend.json:97]: @thisisjp Same as above 🙏

File: status/backend.json (hunk 1 above)
Comment (FrancescaMariaTIberti) [inline status/backend.json:92]:
```suggestion
          "en-EN": "Tell us about your experience with payment and help us improve."
```

File: status/backend.json (hunk 2 above)
Comment (FrancescaMariaTIberti) [inline status/backend.json:97]:
```suggestion
            "en-EN": "Go to survey"
```

[the remark this answers is not in the document]
Comment (Hantex9) [reply status/backend.json]: Addressed by @FrancescaMariaTIberti

File: status/backend.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (Hantex9) [reply status/backend.json]: Addressed by @FrancescaMariaTIberti

**Review by Ladirico — APPROVED** [review]
LGTM! @hevelius your approve is needed to unlock the merge 🙏🏼

**Pull request merged**